### DISCUSSION TITLE
importacion de bibliotecas

### DISCUSSION THREAD

**Discussion by nicolasagosti** [dds-jv/foro#413] [Consultas de herramientas] [answered]
Buenas tardes, quiero saber si se pueden importar bibliotecas como: java.util.Date; y java.util.Scanner.

**Comment by flbulgarelli** [comment, marked as answer]
👋  ¡Hola!

Si, tanto para el TPA como los TPI se puede usar cualquier parte de la biblioteca estándar de Java, con las siguientes consideraciones: https://dds-jv.github.io/pautas/sobre-los-trabajos-practicos.

En particular sobre tu pregunta, nota que ahí encontrarás una explicación sobre por qué no usar `java.util.Date`. 

¡Saludos!
